src/backends/sqlite3/statement.cpp

```diff
@@ -201,25 +201,29 @@ statement_backend::exec_fetch_result
 sqlite3_statement_backend::load_one()
 {
     statement_backend::exec_fetch_result retVal = ef_success;
-
-    int const res = sqlite3_step(stmt_);
-
-    if (SQLITE_DONE == res)
-    {
-        databaseReady_ = false;
+    if( !databaseReady_ )
```

Comment:
Instead of indenting everything in else, can you just write:
``` c++
if (!databaseReady_)
    return ef_no_data;
```

It's easier to read instead of embedded if/else